types/node/fs.d.ts

```diff
@@ -2215,9 +2215,9 @@ declare module 'fs' {
      * If this method is invoked as its `util.promisify()` ed version, it returns
      * a promise for an `Object` with `bytesRead` and `buffer` properties.
      * @since v0.0.2
-     * @param [buffer=Buffer.alloc(16384)] The buffer that the data will be written to.
```

Comment:
Did you change this or was this my script?

Reply:
Script

Reply:
Mhhh, will look into.